src/traces/sankey/render.js

```diff
@@ -836,6 +836,23 @@ module.exports = function(gd, svg, calcData, layout, callbacks) {
         .style('pointer-events', 'auto')
         .attr('transform', sankeyTransform);
 
+    sankey.each(function(d, i) {
+        gd._fullData[i]._sankey = d;
+
+        // Draw dragbox
+        Lib.ensureSingle(gd._fullLayout._draggers, 'rect', 'bg-' + i, function(el) {
```

Comment:
Having something like `bgsankey-' + trace.uid` would be safer.

Note that you'll need to remove this dragger layer in 

https://github.com/plotly/plotly.js/blob/c6573487f33ffe02f38f443690e5fdb824fdb161/src/traces/sankey/base_plot.js#L29-L36

Reply:
Done in 20d8c92d8454cc804e71b84a90469df807d14898 and :lock: down with a test.